src/components/atoms/Cv/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Cv.components';
```

Comment:
Powinniśmy to eksportować pod namespace `Cv` w takim przypadku.

`Cv.Heading` itd.